server/src/microservices/app.service.ts

```diff
@@ -83,7 +85,16 @@ export class AppService {
     await this.metadataService.init();
   }
 
+  async postInit() {
+    await this.libraryService.watchAll();
+  }
+
+  async stopWatcher() {
+    await this.libraryService.unwatchAll();
```

Comment:
why can't this be a normal teardown method?

Reply:
Fixed